mk/top_of_makefile.mk

```diff
@@ -231,7 +230,9 @@ CPPFLAGS += \
 # it's use seems questionable for that kind of target anyway.
 # The launchpad.net arm-none-eabi-gcc toolchain (at least) uses -fshort-enums.
 ifneq ($(TARGET_SYS),none)
-CPPFLAGS += -fstack-protector
+	ifneq ($(findstring redox,$(TARGET_SYS)),redox)
```

Comment:
Again, similar question. Why are we doing:
```make
ifneq ($(findstring redox,$(TARGET_SYS)),redox)
```
instead of:
```make
ifneq ($(TARGET_SYS),redox)
```

We need to know this so that we can do the right thing when we replace the makefile.